src/query/interpreter.rs

```diff
@@ -47,77 +56,69 @@ fn _eval_binary_expr(
     let (right, right_node_type) = if let RuntimeType::Element(ASTNode::PrimarySymbol(val)) = &rhs {
         (val.symbol.as_str(), &val.kind)
     } else {
-        ("", &LiteralType::NONE_TYPE)
+        ("", &LiteralType::NoneType)
     };
 
     match operator {
         "=" => match right_node_type {
-            LiteralType::NUMERIC_LITERAL => RuntimeType::Bool(
-                get_value_from_obj(obj, &left)
+            LiteralType::NumericLiteral => RuntimeType::Bool(
+                get_value_from_obj(obj, left)
                     .to_string()
                     .parse::<f64>()
                     .expect("Not a Number")
                     == right.parse::<f64>().expect("Not a Number"),
             ),
-            _ => {
-                return RuntimeType::Bool(
-                    get_value_from_obj(obj, &left).to_string() == right.to_string(),
-                );
-            }
+            _ => RuntimeType::Bool(get_value_from_obj(obj, left) == right),
```

Comment:
Changing this to 
_ => RuntimeType::Bool(get_value_from_obj(obj, left).to_string() == right.to_string()),
fixes the issue.